src/Jalalian.php

```diff
@@ -648,16 +653,31 @@ public function getNextWeek(): Jalalian
         return $this->addDays(7);
     }
 
+    public function getLastWeek(): Jalalian
+    {
+        return $this->subDays(7);
+    }
+
     public function addDays(int $days = 1): Jalalian
     {
         return static::fromCarbon($this->toCarbon()->addDays($days));
     }
 
+    public function addDay(): Jalalian
+    {
+        return static::fromCarbon($this->toCarbon()->addDays());
```

Comment:
use `addDays(1)` here

Reply:
I see, I wanted it to be like `Carbon` and support the same name, but I can change it.

Reply:
No. I mean use `addDays(1)` instead of `static::fromCarbon($this->toCarbon()->addDays());`

Reply:
Done.